submissions/ALT+F4 LTN-RLD-Cybersyn Mod Comparison/Requesting a Train.md

```diff
@@ -0,0 +1,111 @@
+---
+author: OceanPhantom
+editor: 
+image:
+title:
+lang: en
+discuss:
+  discord: https://alt-f4.blog/discord
+  forum: https://forums.factorio.com/viewtopic.php?f=5&t=98585
+  reddit: https://www.reddit.com/r/factorio/comments/nmxfxn/altf4_37_combinator_crash_course_continuation/
+---
+
+## Mod Spotlight / Mod Comparison / Requesting A Train <author>OceanPhantom</author>
+
+Factorio's train system is great. Trains are the transportation method of choice when it comes to long distances. Its rules are exceedingly simple, allowing you to build your first automated bi-directional rail network to a nearby ore outpost with incredible ease. And yet these rules, taken all together, can make setting up a network of hundreds of trains to feed your megabases's insatiable demand for raw resources a logistic nightmare. 
+
+They allow for such incredible complexity that [u/minibetrayal](https://www.reddit.com/u/minibetrayal/) on reddit theorized that the train system was actually Turing Complete. And then took on the challenge of [building the largest and most useless rail network ever](https://www.reddit.com/r/factorio/comments/evkff6/the_biggest_and_most_useless_rail_network_ever/) to prove it. Making the rail system join the circuit network and the belt transport system ([issue #23](https://alt-f4.blog/ALTF4-23/) and [issue #24](https://alt-f4.blog/ALTF4-24/)) as yet another independently Turing Complete part of Factorio.
+
+With all the options available to us, how could the train system possibly get better? 
```

Comment:
Maybe we can mention here already what's possible with train limits? Up to you though